Setup/Uninstall.php

```diff
@@ -40,21 +41,33 @@ public function __construct(Config $config)
         $this->loggingClient = new LoggingClient();
     }
 
+    /**
+     * Uninstall a single merchant
+     *
+     * @param string $storeId - the id of the individual store getting uninstalled
+     * @param ProtectMetadata $store - the store info
+     */
+    private function uninstallMerchant(string $storeId, ProtectMetadata $store): void
```

Comment:
`storeId` is `int`

Reply:
also, quibble, but we probably want to refer to `ProtectMetadata` instances as `metadata`, since they're not actually `Store` instances

Reply:
> `storeId` is `int`

yeah, I see that now, I got thrown off by the comment: on Helper/Config.php:231 `     * Structured as { [storeId: string]: ProtectMetadata };
`